### PR TITLE
Helen contactscreen

### PR DESCRIPTION
**Pull request by hwang635**
Finished lofi contact screen
- Form takes in user input, submits if all fields are filled (if not filled, alerts with an error message), and airtable automatically sends email with the form input
- Icons lead to APIFM's website, facebook, and instagram

TODO: revisit when jeff's menu is added for small spacing fix and to remove contact screen button

### CODE REVIEW

File: src/ContactScreen.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        value={name}
+        style={styles.textFields}
+      />
+      <TextInput
+        onChangeText={setEmail}
+        placeholder="Email"
+        value={email}
+        style={styles.textFields}
+      />
+      <TextInput
```
Comment (he-is-james) [inline src/ContactScreen.js]: This section should validate that the input is an email
https://reactnative.dev/docs/textinput
`keyboardType="email-address"` is a to have

File: src/ContactScreen.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  },
+  messageTextField: {
+    borderWidth: 1,
+    borderRadius: 10,
+    borderColor: 'grey',
+    fontSize: 14,
+    marginBottom: 15,
+    paddingLeft: 20,
+    height: 130,
+    textAlignVertical: 'top', // prevent text from going to the centre
```
Comment (he-is-james) [inline src/ContactScreen.js]: remove comments

**Review by he-is-james — CHANGES_REQUESTED** [review]
overall, great work!

File: src/ContactScreen.js (hunk 2 above)
Comment (hwang635) [reply src/ContactScreen.js]: this is a useful comment/for future ref lol

Comment (he-is-james) [reply src/ContactScreen.js]: oh whoops lol

**Review by he-is-james — APPROVED** [review]

**Pull request merged**